R/xgx_geom_smooth_emax.R

```diff
@@ -0,0 +1,145 @@
+#' Plot Emax fit to data
+#' Uses nlsLM, predictdf.nls, and stat_smooth to display Emax model fit to data
```

Comment:
write "Uses minpack.lm::nlsLM so that it's clear where it comes from."  I forgot what package was needed and got confused at first when I tried to use it.